recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "tokenizers" %}
-{% set version = "0.19.1" %}
+{% set version = "0.20.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: ee59e6680ed0fdbe6b724cf38bd70400a0c1dd623b07ac729087270caeac88e3
+  sha256: 84edcc7cdeeee45ceedb65d518fffb77aec69311c9c8e30f77ad84da3025f002
   patches:  # [win]
     - remove-multiprocess-fork.patch  # [win]
```

Comment:
the patch removes `mp.set_start_method("fork")` which at first glance looks like enough to allow the test to run on windows. But it also then removes the test and the test is skipped in the test section. Seems like just removing the fork command would be enough to get it to run? Maybe @ViridianMelody could clarify?

Reply:
Removing `test_multiprocessing_with_parallelism` will cause `AttributeError: Can't pickle local object 'multiprocessing_with_parallelism.<locals>.encode'` for these tests:
```
FAILED tests/bindings/test_tokenizer.py::TestTokenizer::test_multiprocessing_with_parallelism
tests/implementations/test_bert_wordpiece.py::TestBertWordPieceTokenizer::test_multiprocessing_with_parallelism
FAILED tests/implementations/test_byte_level_bpe.py::TestByteLevelBPE::test_multiprocessing_with_parallelism
FAILED tests/implementations/test_char_bpe.py::TestCharBPETokenizer::test_multiprocessing_with_parallelism
```